sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -150,28 +160,48 @@ def __init__(self, flags=None, **kwargs):
     arguments and then parse the command line specified by flags or by default
     the one obtained from sys.argv.
 
-    The subclasses are not expected to require a redefinition of __init__.
+    The subclasses of PipelineOptions should not redefine __init__.
```

Comment:
What happens if they define an __init__ and call super's __init__ as well?

Reply:
Things should work. Do you prefer a different wording? `The subclasses of PipelineOptions do not need to redefine __init__` perhaps?

Reply:
Changed this, ptal.

Reply:
New versions looks good, thanks.